test/axes/timeAxisTests.ts

```diff
@@ -294,6 +294,34 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels show correctly when display format is set to 'center'", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    scale.domain([new Date("2015-09-02"), new Date("2015-09-03")]);
+    axis.tierLabelPositions(["center", "center"]);
+    axis.renderTo(svg);
+
+    let labels = axis.content().selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`);
+    assert.notStrictEqual(labels.size(), 0, "More than one labels are selected in testing");
```

Comment:
I think I messed up in my review.  Especially since the message reports information about using "more than one labels", the assert should reflect as such.

Thus, can we change this to...
`assert.operator(labels.size(), ">", 1, msg)`